### PR TITLE
[WIP] Handle JSON Parse Error in OpenAPI documentation tab

### PR DESCRIPTION
**Pull request by SafarMirek**
This pr is adding error screen in documentation tab when JSON parsing of OpenAPI fails. Fixes issue #1624.

There should be discussion about the displayed text.

apicurio-document-tab-error

### CODE REVIEW

**Comment by EricWittmann** [comment]
Nice work, thanks @SafarMirek 

Is this ready for final review?  (it's marked as a Draft PR still)

**Comment by SafarMirek** [comment]
If you like the displayed error text on the page, it's ready.

File: ui/src/app/pages/artifactVersion/components/tabs/errorTab.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    }
+
+    // Error description can be in node children - if it is missing, default description is used
+    private errorDescription(): ReactNode {
+        if (this.props.children) {
+            return this.props.children;
+        } else {
+            return (
+                "Something went seriously wrong on the server.  We'll try to provide you with some more information about " +
+                "the problem (see below)."
```
Comment (jsenko) [inline ui/src/app/pages/artifactVersion/components/tabs/errorTab.tsx:113]: I'd leave the word `seriously` from the error message, unless the server is on fire:)

**Review by jsenko — APPROVED** [review]
I left a small suggestion, otherwise looks good!

File: ui/src/app/pages/artifactVersion/components/tabs/errorTab.tsx (hunk 1 above)
Comment (EricWittmann) [reply ui/src/app/pages/artifactVersion/components/tabs/errorTab.tsx:113]: I'll update the text of this. :)

**Comment by EricWittmann** [comment]
Nice work, thanks @SafarMirek

**Pull request merged**